### PR TITLE
get upstream IPs

### PR DESCRIPTION
**Pull request by malizz**
### Description
Get Upstream IPs and clusters for troubleshooting tool 

### PR Checklist

* [x] updated test coverage
* [ ] external facing docs updated
* [x] not a security concern

### CODE REVIEW

File: troubleshoot/proxy/upstreams.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
+package troubleshoot
```
Comment (ndhanushkodi) [inline troubleshoot/proxy/upstreams.go:1]: Should these upstreams files be in troubleshoot/upstreams? I figured the structure would be something like 

```
troubleshoot
  proxy
    *files related to the troubleshoot proxy command*
  upstreams
    *files related to the troubleshoot upstreams command* 
```

**Review by ndhanushkodi — APPROVED** [review]
Looks great Mali, just had a question about the dir/package structure!

File: troubleshoot/proxy/upstreams.go (hunk 1 above)
Comment (malizz) [reply troubleshoot/proxy/upstreams.go:1]: We use the same receiver in those files, separating them needs a refactor, will create a separate PR for that.

**Review by curtbushko — APPROVED** [review]
LGTM! Thanks for putting this together!

**Pull request merged**